rpmconf/Install/Util/utilfunc.sh

```diff
@@ -375,6 +375,27 @@ checkRecentBackup() {
   fi
 }
 
+checkCAKeyLength() {
+	isInstalled "zimbra-ldap"
+	if [ x$PKGINSTALLED != "x" ]; then
+		openssl_cmd=""
+		keyfile="/opt/zimbra/conf/ca/ca.key"
+		if [[ -x "/opt/zimbra/common/bin/openssl" ]]; then
```

Comment:
in which case /opt/zimbra/common/bin/openssl will not be present?

Reply:
@silentsakky it may be for old zcs versions 8.0 or earlier.